packages/website/package.json

```diff
@@ -38,7 +38,7 @@
         "less": "^2.7.2",
         "lodash": "^4.17.4",
         "material-ui": "^0.17.1",
-        "moment": "^2.18.1",
+        "moment": "2.13.0",
```

Comment:
What's the reason for the downgrade? I believe latest moment.js now includes types in the package.

Reply:
Yep, we can also consolidate versions to a higher one. Going with `2.21.0` since thats what another dep uses, so we only include a single version :)